cypress/commons/TestConstants.js

```diff
@@ -50,6 +46,12 @@ export const PAGE_NAME = {
   SIGN_IN: '/sign-in'
 };
 
+export const URL_REGEX = {
```

Comment:
"WITHOUT_SUFFIX", "WITH_UNKNOWN_ID_SCENARIO_SUFFIX" and "WITH_RUN_SUFFIX" can be more specific. Could you please rename these constants ?